dapps/organizations/src/components/detail/identifications/notary/notary.identifications.ts

```diff
@@ -76,11 +75,11 @@ function triggerRequestReload(orgAddress: string, detail: any) {
  * Return the list of requested identifications
  */
 async function getRequests(runtime: bcc.Runtime, address: string) {
-  const evanAuthHeader = await generateEvanAuthHeader(runtime);
+  const evanAuthHeader = await bcc.utils.getSmartAgentAuthHeaders(runtime);
   const allRequests = await axios({
     method: 'POST',
     url: `${ agentUrl }/api/smart-agents/smart-agent-2fi/status/getAll`,
-    headers: { 'Authorization': evanAuthHeader }
+    headers: { 'Authorization': await bcc.utils.getSmartAgentAuthHeaders(runtime), },
```

Comment:
wtf? you generate the auth header above
```suggestion
    headers: { 'Authorization': evanAuthHeader },
```

Reply:
removed `const evanAuthHeader = await generateEvanAuthHeader(runtime);`